aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -326,7 +327,6 @@ impl AptosVM {
                 ) {
                     return discard_error_vm_status(e);
                 }
-                let fee_statement = AptosVM::fee_statement_from_gas_meter(txn_data, gas_meter);
```

Comment:
For a moment I thought this would contain different numbers than the epilogue saw since the epilogue.